bootstrap.py

```diff
@@ -61,190 +69,109 @@
                     help="Comma-separated module name hierarchies whose log "
                          "messages should be shown. e.g., "
                          "spyder.plugins.completion,spyder.plugins.editor")
+parser.add_argument('--no-subrepos', action='store_true', default=False,
```

Comment:
```suggestion
parser.add_argument('--no-install', action='store_true', default=False,
```

The key point is not that subrepos are not _used_, but rather that they are not _installed_ automatically, so updated the CLI arg name to clarify that

Reply:
Agreed.